src/login/oidc/redirectHandler/TokenSaver.ts

```diff
@@ -43,43 +42,33 @@ export default class TokenSaver implements ITokenSaver {
   ) {}
 
   async saveTokenAndGetSession(
-    localUserId: string,
+    sessionId: string,
     idToken: string,
     accessToken?: string,
     refreshToken?: string
-  ): Promise<ISessionInfo> {
-    throw new Error("Not Implemented");
-    // const decoded = await this.joseUtility.decodeJWT(
-    //   // TODO this should actually be the id_vc of the token
-    //   accessToken as string
-    // );
-    // // TODO validate decoded token
-    // // TODO extract the localUserId from state and put it in the session
+  ): Promise<void> {
+    // throw new Error("Not Implemented");
```

Comment:
No need to keep this anymore.

Reply:
Fixed in e65ae11bb5e81297c1fbc3e19e0219f4d3b5282d